[2022-02-15 03:05:32,842] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.model.my_new_project.stg_customers manual__2022-02-15T03:04:57.520087+00:00 [queued]>
[2022-02-15 03:05:32,856] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.model.my_new_project.stg_customers manual__2022-02-15T03:04:57.520087+00:00 [queued]>
[2022-02-15 03:05:32,857] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 03:05:32,857] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-15 03:05:32,858] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 03:05:32,872] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): model.my_new_project.stg_customers> on 2022-02-15 03:04:57.520087+00:00
[2022-02-15 03:05:32,880] {standard_task_runner.py:52} INFO - Started process 454 to run task
[2022-02-15 03:05:32,885] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dbt_training_bytecode', 'model.my_new_project.stg_customers', 'manual__2022-02-15T03:04:57.520087+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/dbt-***.py', '--cfg-path', '/tmp/tmphxoto2l4', '--error-file', '/tmp/tmpe3qw6hnl']
[2022-02-15 03:05:32,887] {standard_task_runner.py:77} INFO - Job 91: Subtask model.my_new_project.stg_customers
[2022-02-15 03:05:33,018] {logging_mixin.py:109} INFO - Running <TaskInstance: dbt_training_bytecode.model.my_new_project.stg_customers manual__2022-02-15T03:04:57.520087+00:00 [running]> on host 459d09dd7c86
[2022-02-15 03:05:33,158] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bytecode
AIRFLOW_CTX_DAG_ID=dbt_training_bytecode
AIRFLOW_CTX_TASK_ID=model.my_new_project.stg_customers
AIRFLOW_CTX_EXECUTION_DATE=2022-02-15T03:04:57.520087+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-15T03:04:57.520087+00:00
[2022-02-15 03:05:33,161] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-15 03:05:33,163] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            dbt --no-write-json run --target dev --models stg_customers             --profiles-dir /opt/***/dbt/dbt-training-bytecode-1 --project-dir /opt/***/dbt/dbt-training-bytecode-1\n            ']
[2022-02-15 03:05:33,203] {subprocess.py:85} INFO - Output:
[2022-02-15 03:05:41,399] {subprocess.py:89} INFO - 03:05:41  Running with dbt=1.0.1
[2022-02-15 03:05:42,011] {subprocess.py:89} INFO - 03:05:42  Found 8 models, 11 tests, 0 snapshots, 0 analyses, 367 macros, 0 operations, 1 seed file, 3 sources, 0 exposures, 0 metrics
[2022-02-15 03:05:42,017] {subprocess.py:89} INFO - 03:05:42
[2022-02-15 03:05:45,326] {subprocess.py:89} INFO - 03:05:45  Concurrency: 4 threads (target='dev')
[2022-02-15 03:05:45,329] {subprocess.py:89} INFO - 03:05:45
[2022-02-15 03:05:45,347] {subprocess.py:89} INFO - 03:05:45  1 of 1 START view model dbt_bytecode_training.stg_customers..................... [RUN]
[2022-02-15 03:05:46,906] {subprocess.py:89} INFO - 03:05:46  1 of 1 OK created view model dbt_bytecode_training.stg_customers................ [SUCCESS 1 in 1.54s]
[2022-02-15 03:05:46,959] {subprocess.py:89} INFO - 03:05:46
[2022-02-15 03:05:46,961] {subprocess.py:89} INFO - 03:05:46  Finished running 1 view model in 4.94s.
[2022-02-15 03:05:46,966] {subprocess.py:89} INFO - 03:05:46
[2022-02-15 03:05:46,968] {subprocess.py:89} INFO - 03:05:46  Completed successfully
[2022-02-15 03:05:46,973] {subprocess.py:89} INFO - 03:05:46
[2022-02-15 03:05:46,976] {subprocess.py:89} INFO - 03:05:46  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2022-02-15 03:05:47,435] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-15 03:05:47,468] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dbt_training_bytecode, task_id=model.my_new_project.stg_customers, execution_date=20220215T030457, start_date=20220215T030532, end_date=20220215T030547
[2022-02-15 03:05:47,517] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-15 03:05:47,575] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
